crates/matrix-sdk-common/src/tracing_timer.rs

```diff
@@ -109,8 +109,14 @@ mod tests {
     async fn test_timer_name() {
         use tracing::{span, Level};
 
+        if std::env::var("RUST_LOG").is_err() {
+            std::env::set_var("RUST_LOG", "trace");
```

Comment:
Nononono, not `set_var`!¹ `tracing_subscriber` has a rich enough API that falling back to a different string when the env var is not set is easy enough: https://github.com/tokio-rs/axum/blob/main/examples/tracing-aka-logging/src/main.rs#L23-L32

¹ https://github.com/rust-lang/rust/issues/90308

Reply:
It's a test-only, but alright.